kafka-client/src/main/java/dev/responsive/kafka/store/CommitBuffer.java

```diff
@@ -225,12 +235,61 @@ int size() {
     return buffer.size();
   }
 
-  public void flush(long offset) {
-    if (buffer.size() < flushThreshold) {
-      LOG.debug("Ignoring flush() - commit buffer {} smaller than flush threshold {}",
+  private long totalBytesBuffered() {
+    return buffer.values().stream()
+        .mapToLong(e -> plugin.bytes(e.key).get().length + (e.isTombstone ? 0 : e.value.length))
+        .sum();
+  }
+
+  private boolean shouldFlush() {
+    boolean recordsTrigger = false;
+    boolean bytesTrigger = false;
+    boolean timeTrigger = false;
+    if (buffer.size() >= flushTriggers.getRecords()) {
+      LOG.info("will flush due to records buffered {} over trigger {}",
```

Comment:
let's include store name + partition in all of these logs (my log format is `tableName[kafkaPartition:subpartition]` where `subpartition` is optional)

either that or migrate to AK's prefixed log class

Reply:
💯  the log prefix is great we should use that all the time where there's going to consistent relevant context (like name of store, partition, thread, etc)